### PR TITLE
rmda user-manual

### PR DESCRIPTION
**Pull request by ferris-cx**
Ⅰ. Describe what this PR does
Since Gpus in AI scenarios require RDMA computing nics for high-speed NCCL communication, end-to-end support for rdma devices must be added, including device discovery, device registration, node resource update, scheduling, and allocation.
Ⅱ. Does this pull request fix one issue?
No
Ⅲ. Describe how to verify it
Ⅳ. Special notes for reviews
V. Checklist
I have written necessary docs and comments
I have added necessary unit tests and integration tests
All checks passed in make test

### CODE REVIEW

File: docs/best-practices/gpu-rdma-joint-allocation.md [hunk 1]
```diff
@@ -0,0 +1,1219 @@
...
+    "ipam": {
+      "type": "host-local",
+      "subnet": "10.20.12.0/24",
+      "rangeStart": "10.20.12.121", //Plan the IP address range of the Pod
+      "rangeEnd": "10.20.12.121"
+    }
+  }'
+  ```
+
+Plan: Nad configuration file name of NIC ens3f0np0 on node2: sriov-attach-k8s-node2-ens3f0np0-kubeflow-conf.yaml.
```
Comment (ZiMengSheng) [inline docs/best-practices/gpu-rdma-joint-allocation.md:447]: NAD 的安装应该放在 multus 安装部分？

File: docs/best-practices/gpu-rdma-joint-allocation.md [hunk 2]
```diff
@@ -0,0 +1,1219 @@
...
+```
+
+If it is displayed that you can directly jump to the inside of another pod-vf02 container, it means that the no-secret setting is successful!
+
+##### Double machine IB communication test
+
+1* GPU1+1* RDMA communication between two Pods 2G data volume communication scenario
+
+```shell
+mpirun --allow-run-as-root -H 10.244.1.10:1,10.244.2.21:1 -mca plm_rsh_args "-p 20024" -x NCCL_IB_DISABLE=0 -x NCCL_DEBUG=INFO -x NCCL_SOCKET_IFNAME=eth0 -x NCCL_IB_HCA==mlx5_2 -x UCX_NET_DEVICES=eth0 -x NCCL_NET_GDR_READ=1 ./build/all_reduce_perf -b 2M -e 2G -f 2 -g 1 -n 100 -w 5
```
Comment (ZiMengSheng) [inline docs/best-practices/gpu-rdma-joint-allocation.md:1155]: please give a brief intruduction about mpirun

File: docs/best-practices/gpu-rdma-joint-allocation.md [hunk 3]
```diff
@@ -0,0 +1,1219 @@
+## A test report on affinity scheduling of rdma nic and GPU on k8s and high speed communication of RDMA computing network
```
[the remark this answers is not in the document]
Comment (ferris-cx) [reply docs/best-practices/gpu-rdma-joint-allocation.md:1]: oK。我改下标题，尽量简短

File: docs/best-practices/gpu-rdma-joint-allocation.md (hunk 1 above)
Comment (ferris-cx) [reply docs/best-practices/gpu-rdma-joint-allocation.md:447]: 是的，NAD跟multus耦合的

**Pull request closed without merging**